From 5629d05c4facaed2f6fdc4c619bb4fde02e9f6a3 Mon Sep 17 00:00:00 2001
From: Marc-Antoine Perennou <Marc-Antoine@Perennou.com>
Date: Mon, 23 May 2016 16:10:41 +0200
Subject: [PATCH] build-sys: update supported ruby versions

add ruby 2.3
default to ruby 2.3
drop 2.0 and 1.9 which are dead

Signed-off-by: Marc-Antoine Perennou <Marc-Antoine@Perennou.com>
---
 configure.ac | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/configure.ac b/configure.ac
index 4e808d8..d1a38cc 100644
--- a/configure.ac
+++ b/configure.ac
@@ -635,12 +635,12 @@ AC_MSG_RESULT($enable_ruby)
 
 if test "x$enable_ruby" = "xyes" ; then
 	AC_ARG_WITH([ruby-version],
-		AS_HELP_STRING([--with-ruby-version=2.2], [Specify Ruby version]),
+		AS_HELP_STRING([--with-ruby-version=2.3], [Specify Ruby version]),
 		RUBY_VERSION="$withval",
-		RUBY_VERSION="2.2")
+		RUBY_VERSION="2.3")
 
 	case "$RUBY_VERSION" in
-		1.9|2.0|2.1|2.2)
+		2.1|2.2|2.3)
 			;;
 		*)
 			AC_MSG_ERROR([Invalid ruby version specified])
-- 
2.8.3

